cirq-core/cirq/json_resolver_cache.py

```diff
@@ -156,6 +156,7 @@ def _parallel_gate_op(gate, qubits):
         'VarianceStoppingCriteria': cirq.work.VarianceStoppingCriteria,
         'VirtualTag': cirq.VirtualTag,
         'WaitGate': cirq.WaitGate,
+        'QasmUGate': cirq.circuits.qasm_output.QasmUGate,
```

Comment:
Should have alphabetical ordering (before `'QuantumFourierTransformGate': cirq.QuantumFourierTransformGate,`. While we are at it, let's fix the ordering of `'RandomGateChannel': cirq.RandomGateChannel,` as well.

Reply:
It is more than that! Many items are placed in the wrong order. Since you mentioned it, I reordered them in the new commit.

Reply:
Nice! Thanks for going the extra mile! I've left a few minor comments then this should be good to merge.